examples/tinywl-new/wallpaperimage.cpp

```diff
@@ -0,0 +1,100 @@
+// Copyright (C) 2024 WenHao Peng <pengwenhao@uniontech.com>.
+// SPDX-License-Identifier: Apache-2.0 OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+
+#include "wallpaperprovider.h"
+#include "wallpaperimage.h"
+#include "helper.h"
+
+WallpaperImage::WallpaperImage(QQuickItem *parent)
+    :QQuickImage(parent)
+{
+    auto provider = Helper::instance()->qmlEngine()->wallpaperImageProvider();
+    connect(provider, &WallpaperImageProvider::wallpaperTextureUpdate, this, &WallpaperImage::updateWallpaperTexture);
+}
+
+WallpaperImage::~WallpaperImage()
+{
+
+}
+
+QString WallpaperImage::userId()
+{
+    return m_userId;
+}
+
+void WallpaperImage::setUserId(const QString& name)
```

Comment:
Is it a `pid`? pid's type should is int, if is user name, this function name should is `setUserName`